site/content/en/docs/Reference/fleetautoscaler.md

```diff
@@ -177,6 +177,125 @@ spec:
 See the [Webhook Endpoint Specification](#webhook-endpoint-specification) for the specification of the incoming and 
 outgoing JSON packet structure for the webhook endpoint.
 
+## Schedule and Chain Autoscaling
+
+A Schedule based autoscaler can be used to autoscale `GameServers` based on a schedule associated with a policy.
```

Comment:
Nit: "A scheduled based ... based on a schedule" could be reworded. Something like: A schedule-based autoscaler can automatically scale GameServers at a specific time based on a FleetAutoscaler policy.